app/src/main/java/com/procleus/brime/SigninActivity.java

```diff
@@ -183,6 +187,23 @@ public void onErrorResponse(VolleyError error) {
 
     }
 
+    public boolean cvalidate(String uid,String pass) {
```

Comment:
Use a more sensible name like, `clientValidation`
The function name should make sense. Also use camel cases